media-sound/drumkv1/drumkv1-9999.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="An old-school all-digital drum-kit sampler synthesizer with stereo fx"
+HOMEPAGE="http://drumkv1.sourceforge.net/"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://download.sourceforge.net/${PN}/${P}.tar.gz"
```

Comment:
Good point that you mention this. It was actually something I wanted to discuss.

I started with the releases on GitHub, but they are just GitHub's auto generated tarballs, which means no `./configure` script, so you have to run `eautoreconf` just like when cloning the project.
I then switched to sourceforge because the releases there did have a `./configure` script.

Now I was wondering what we'd prefer to do:
- Use the sourceforge mirror and only call `eautoreconf` for the `9999` ebuild
- Use GitHub's release and always call `eautoreconf`